Cargo.toml

```diff
@@ -3,18 +3,21 @@ name = "netconf-rs"
 version = "0.2.6"
 authors = ["Jiajie Chen <c@jia.je>"]
 description = "A Rust library for NETCONF(RFC 2641)"
+readme = "README.md"
+repository = "https://github.com/jiegec/netconf-rs"
 license = "MIT"
 edition = "2021"
+rust-version = "1.75.0"
```

Comment:
According to [the rust-version field documentation](https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field), it is the minimum rust version required to build this crate, which is too new. I think we can ignroe this.

Reply:
I saw that too. Given my understanding of how Rust handles features by asserting this minimum, we can safely assume no need for the nightly branch. It is unknown to me whether any nightly features were actually required, but if they are, we need not build on that tool chain.